packages/form-validations/index.html

```diff
@@ -0,0 +1,35 @@
+{{#> layout/default }}
+
+<script type="module">
+  import FormValidations from './src/index.ts';
+
+  window.onload = function () {
+    var form = document.getElementById("form1");
+
+    // create the form validations instance
+    var formValidations = new FormValidations(form);
+
+    form.addEventListener('submit', function (event) {
+      event.preventDefault();
+
+      // check if the form is valid
+      var valid = formValidations.validate(); // returns true or false
+    });
+  };
+</script>
+
+<div class="docs-FormFieldGrid">
+
+  <form id="form1" method="get" novalidate>
+    <div class="Stack">
+      <div class="TextField" data-spirit-validate>
+        <label for="textfieldSimple" class="TextField__label TextField__label--required">Label</label>
+        <input type="text" id="textfieldSimple" class="TextField__input" placeholder="Placeholder" required />
+      </div>
+      <input type="submit" value="Submit" class="Button Button--primary Button--medium" />
```

Comment:
can we avoid the input type submit and use <button>? It is ugly and cannot have children and html5 way is using the button.

Reply:
<iframe src="https://giphy.com/embed/ehA575gOh0RIQ" width="480" height="439" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/mustache-ehA575gOh0RIQ">via GIPHY</a></p>

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit